frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseMergeParentIntoBranchAction.java

```diff
@@ -55,6 +56,17 @@ public List<SyncToParentStatus> getEligibleStatuses() {
   protected void onUpdate(AnActionEvent anActionEvent) {
     super.onUpdate(anActionEvent);
     syncToParentStatusDependentActionUpdate(anActionEvent);
+    val presentation = anActionEvent.getPresentation();
```

Comment:
Okay, still the same point... tl;dr how was/would the behavior be wrong if it weren't for this change?

Reply:
from a context menu, you can pick the currently checked out branch ofc. In that case, the action name "Checkout and do smth" sounds silly and is inaccurate b/c the branch is already checked out

Reply:
Awww okay 🤦🏻